src/routing/message_accumulator.rs

```diff
@@ -0,0 +1,10 @@
+use bls_signature_aggregator::{Error, Proof, ProofShare, SignatureAggregator};
```

Comment:
don't think we need to intruduce this struct? 
also, the copy right text is missed?